PWGCF/EbyEFluctuations/Tasks/netprotonCumulantsMc.cxx

```diff
@@ -960,6 +1013,36 @@ struct NetprotonCumulantsMc {
     histos.fill(HIST("hgenProfileProton"), cent, nProt);
     histos.fill(HIST("hgenProfileAntiproton"), cent, nAntiprot);
     genEbyeCollisions(cent, netProt, nProt, nAntiprot);
+
+    // Profiles for generated level cumulants
+    //-------------------------------------------------------------------------------------------
+
+    if (cfgIsCalculateCentral) {
+      histos.get<TProfile>(HIST("GenProf_mu1_netproton"))->Fill(cent, std::pow(netProt, 1.0));
+      histos.get<TProfile>(HIST("GenProf_mu2_netproton"))->Fill(cent, std::pow(netProt, 2.0));
+      histos.get<TProfile>(HIST("GenProf_mu3_netproton"))->Fill(cent, std::pow(netProt, 3.0));
+      histos.get<TProfile>(HIST("GenProf_mu4_netproton"))->Fill(cent, std::pow(netProt, 4.0));
+      histos.get<TProfile>(HIST("GenProf_mu5_netproton"))->Fill(cent, std::pow(netProt, 5.0));
+      histos.get<TProfile>(HIST("GenProf_mu6_netproton"))->Fill(cent, std::pow(netProt, 6.0));
+      histos.get<TProfile>(HIST("GenProf_mu7_netproton"))->Fill(cent, std::pow(netProt, 7.0));
+      histos.get<TProfile>(HIST("GenProf_mu8_netproton"))->Fill(cent, std::pow(netProt, 8.0));
+    }
+
+    if (cfgIsCalculateError) {
+
+      float l_Random = fRndm->Rndm();
```

Comment:
In the case of your `fQXY_Z` is justified not to follow `linter` recommendations but not in this case